libpod/runtime.go

```diff
@@ -395,7 +395,7 @@ func makeRuntime(runtime *Runtime) (err error) {
 	}
 	if !foundRuntime {
 		return errors.Wrapf(ErrInvalidArg,
-			"could not find a working runc binary (configured options: %v)",
+			"could not find a working binary (configured options: %v)",
```

Comment:
Don't we have a path to the runtime?

Reply:
We get a bunch of candidate paths in the config file. This error happens if none of them exist.